src/Application/Components/NewsRoom/NewsRoom.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+const NewsRoom = React.createClass({
+  propTypes: {
+    articles: React.PropTypes.array.isRequired,
+  },
+
+  render() {
+    let count = 0;
+    return (
+      <div id="news-zone">
+        <h3>News Room</h3>
+        <div id="news-queue">
+          <ul>
+            {this.props.articles.map(function (article) {
+              count++;
+              return <li key={count}>{article}</li>;
+            })}
```

Comment:
anything this nested or complex (low bar for complexity i know) can easily and beneficially be extracted into a `renderArticles()` method.